docs/user/intro/docusaurus.rst

```diff
@@ -0,0 +1,84 @@
+
+Docusarus
+=========
+
+.. meta::
+   :description lang=en: Hosting Docusaurus sites on Read the Docs.
+
+`Docusaurus`_ is a static-site generator that builds a single-page application with fast client-side navigation and out-of-the-box documentation features.
+
+Minimal configuration required to build a Docusaurus project on Read the Docs looks like this,
+specifying a nodejs toolchain on Ubuntu, using multiple :ref:`build <config-file/v2:build>` commands to install the requirements,
+build the site, and copy the output to $READTHEDOCS_OUTPUT:
+
+.. code-block:: yaml
+   :caption: .readthedocs.yaml
+
+    version: 2
+    build:
+    os: "ubuntu-22.04"
+    tools:
+        nodejs: "18"
+    commands:
+        # "docs/" was created following the Docusaurus tutorial:
+        # npx create-docusaurus@latest docs classic
+        # but you can just use your existing Docusaurus site
+        #
+        # Install Docusaurus dependencies
+        - cd docs/ && npm install
+        # Build the site
+        - cd docs/ && npm run build
+        # Copy generated files into Read the Docs directory
+        - mkdir --parents $READTHEDOCS_OUTPUT/html/
+        - cp --recursive docs/build/* $READTHEDOCS_OUTPUT/html/
```

Comment:
Once we merge and deploy https://github.com/readthedocs/readthedocs.org/pull/11710 this would be as following:

```yaml
build:
  install:
    - cd docs/ && npm install
  build:
    - cd docs/ && npm run build
  post_build:
    - mkdir --parents $READTHEDOCS_OUTPUT/html/
    - cp --recursive docs/build/* $READTHEDOCS_OUTPUT/html/
```